templates/default/apache2.conf.erb

```diff
@@ -18,6 +18,7 @@
   Alias /stylesheets /etc/<%= node['nagios']['server']['vname'] %>/stylesheets
 <% end -%>
   ScriptAlias <%= node['nagios']['cgi-path'] %> <%= node['nagios']['cgi-bin'] %>
+  ScriptAlias /cgi-bin/statusjson.cgi <%= node['nagios']['cgi-bin'] %>/statusjson.cgi
```

Comment:
Would it not be better to do:
```
ScriptAlias <%= node['nagios']['cgi-path'] %>/statusjson.cgi <%= node['nagios']['cgi-bin'] %>/statusjson.cgi
```
Because it feels like hardcoding /cgi-bin is not very handy, but I can be wrong here....